### PR TITLE
fix(events): fix display for events new_autoscaler and edit_autoscale…

### LINKED ISSUE #681: [events] Add support for `edit_autoscaler`
And make sure other autoscaler-related events are supported too. Doc to add support for new events: ...

### LINKED ISSUE #682: [events] Add support for `repeated_crash`
For instance I noticed the following message when using the `timeline` command:

```
* Wed Jul 21 2021 13:02:36 - Repeated Crash - Unknown event repeated_crash on app my-app <scalingo-platform (deploy@scalingo.com)>
```

The sentence `Unknown event repeated_crash...` is probably due to an unsupported event?

### PR DESCRIPTION
**Pull request by stopostit**
…r in timeline

Fix #681 
Fix #682

- [x] Add a changelog entry in the section "To Be Released" of CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+* fix(events): fix display for events new_autoscaler and edit_autoscaler in timeline [#684](https://github.com/Scalingo/cli/pull/684)
```
Comment (EtienneM) [inline CHANGELOG.md:3]: There is also the `repeated_crash` event that we now support. And this changelog entry should be mve in the "To be Released section

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (stopostit) [reply CHANGELOG.md:3]: Is it possible to link 2 issues to a single PR ?
Like "Fix #681 and #682"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
 # Changelog
 
+
```
Comment (EtienneM) [inline CHANGELOG.md:3]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
 ### To be Released
 
 * feat(deploy): add --no-follow flag to detach deployment logs [#679](https://github.com/Scalingo/cli/pull/679)
 * build(deps): bump github.com/fatih/color from 1.12.0 to 1.13.0 [#680](https://github.com/Scalingo/cli/pull/680)
+* fix(events): fix display for events new_autoscaler edit_autoscaler and repeated_crash in timeline [#684](https://github.com/Scalingo/cli/pull/684)
```
Comment (EtienneM) [inline CHANGELOG.md:8]:
```suggestion
* fix(events): fix display for events new_autoscaler, edit_autoscaler and repeated_crash in timeline [#684](https://github.com/Scalingo/cli/pull/684)
```

**Review by EtienneM — APPROVED** [review]

**Pull request merged**